### PR TITLE
Scripts docs

### PR DESCRIPTION
**Pull request by vetinari**
Add lua scripting docs for gh-pages

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: docs/scripts.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+# Lua filter scripts
+
+[LUA](https://www.lua.org/) scripts can be used as filters in skipper. The
+current implementation supports [Lua 5.1](https://www.lua.org/manual/5.1/).
+
+## Route filters
+
+The lua scripts are set up as filter with the `lua()` filter in a route
```
Comment (aryszka) [inline docs/scripts.md]: The lua scripts can be added to a route description with the `lua()` filter.

**Comment by aryszka** [comment]
:+1:

**Comment by vetinari** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**